filelog_unix.go

```diff
@@ -0,0 +1,5 @@
+// +build !windows
+
+package log4go
+
+const FILELOG_ARCHIVE_REGEX = "\\.[0-9]{4}-[0-9]{2}-[0-9]{2}(\\.[0-9]{4})?(\\.gz)?$"
```

Comment:
That's right - we don't support archiving files with the number-based rotation scheme. Its actually impossible to archive old number-based files because log4go will choose to start over at 1. For example:

```
logfile.log.001
logfile.log.002
logfile.log.003
```

1 is the oldest, 3 is the newest. If we delete 1 or 2, log4go will rotate the next log at `logfile.log.001` (instead of `logfile.log.004`), and the logs will be out of order. On the next age-off, we'd delete the newest file instead of the oldest.

Plus, we only have 999 slots and eventually run out, especially if rotate at startup is enabled. The number-based scheme overall is a 👎